### DISCUSSION TITLE
🚀 Hey Devs, Get Ready for Next-Level Lessons and Juicy Updates! Code On! 🤓

### DISCUSSION THREAD

**Discussion by cromewar** [Cyfrin/foundry-full-course-cu#973] [Announcements]
Yo, how you've been doing? 👋

We're super thrilled to announce that we've taken your feedback seriously and are now expanding our course lessons! and also creating a lot of sick content to expand of the topics you are most interested in! This reference page is your go-to guide for what's new, what's updated, and what you can expect in each lesson. 🌟

We are also adding this a file on the root of the repo so you can see the changes we are making to the course. you can refer to [content-updates.md](https://github.com/Cyfrin/foundry-full-course-f23/blob/main/content-updates.md) file.

## 🎥 New Related Content

- **What to do after finishing Patricks Course**

     So, you've crushed Patrick's course, huh? 🎉 And now you're like, "Cool, but now what?" Well, don't sweat it. This video 🎥 will guide you through 5 ideas of what to do next.

     [Check out the video here!](https://youtu.be/HRW7G0xo8yw?si=op1UuZuGMNDUe1Ni)

## 🕒 Upcoming Content

0️⃣ Solving common problems and erorrs most students face, like outdated packages, libraries and commands that work differently now, etc. This will be a video series that will be released in the coming weeks.

1️⃣ "Building a Charismatic Web3 Resume" 📝

Ready to flex those resume muscles? We're gonna show you how to make that Web3 resume of yours pop! Oh, and we've got some industry hotshots sharing their pro tips. Don't sleep on this!

2️⃣ "Get Hackthon-Ready, Folks!" 🏆

Wanna know how to crush it in those blockchain hackathons? We're talkin' mindset, strategies, and maybe even a quick zen moment before go-time.

3️⃣ "DIY Blockchain Time!" 🛠️

Ever thought about building your own blockchain? Yeah, you heard that right! We're breaking it all down for you, step by step. Layers 1 and 2? We got this.

4️⃣ "Junior Dev Roles in Solidity: Where Are They?" 👨‍💻

Struggling to find junior roles? We're here to help you sniff 'em out. 
> Bonus: We've got interviews lined up with HR peeps and devs who recently jumped into the game. 🔥

5️⃣ "The Fresher's Guide to a Web3 Internship/Job" 🎒

Hey newbies, we got you! We'll show you how to break through the '2 years of experience' wall. And oh, we'll feature some kick-butt stories from peeps who snagged their first Web3 gigs!

6️⃣ "Bot Mania: What Are These Bots Up To?" 🤖

Bots, bots, bots! Let's figure out what they're all about and how you can outwit them. All the while keeping it 💯 legit.

7️⃣ "The Must-Read Web3 Books: Get Your Read On!" 📚

Calling all book nerds! We're gonna dish on the books you can't afford to miss if you're serious about Web3. Might even chat with a few authors, who knows?

8️⃣ "Mastering In-line Assembly and Yul" 🤖

Ready to go under the hood of EVM? Let's talk low-level programming, bit operations, and all that geeky stuff.

## 📚 Updated Lessons

- Content expansion for lesson coming soon!

## 🗓️ Rollout Date

We'll start rolling out these new features and updates from the 1st of next month. But we will update you with some content this month as well.

## 💬 Your Feedback Matters

We're always looking to improve, and your feedback is crucial for that. If you have any suggestions or run into any issues, please don't hesitate to raise an issue right here on GitHub or contact us through Discord.

Happy Learning!

Vasiliy Gualoto & The Cyfrin Team 🌟

**Comment by 0xusmanf** [comment]
I can't wait to see this remarkable upcoming content. 👀 

**HUGE** thanks to the Cyfrin team for bringing this incredible content to us ❤️.

LFG 🚀🚀🚀 🔥🔥🔥

**Comment by sunnyStefi** [comment]
Yess! You guys are totally smashing it 💪🏻 💪🏻 💪🏻

**Comment by vogue20033** [comment]
Can't wait to see the next up date.

**Comment by AAYUSH-GUPTA-coder** [comment]
Amazing !!!! Super excited about Mastering In-line Assembly and Yul.

**Comment by shikhar229169** [comment]
Let's gooooo 🚀🚀🚀🚀

**Comment by theirrationalone** [comment]
Getting ready my fingers again to hammer on my keyboard again.......... 😍😍😍🤩🤩🤩

**Comment by dodeyeikoi** [comment]
Here's my feedback, you guys are the best!!! I'm currently in lesson 9 of the course and I can't wait to enjoy all these upcoming contents
